.release-plz.toml

```diff
@@ -0,0 +1,11 @@
+[workspace]
+git_release_name = "{{ package }}-{{ version }}"
+git_tag_name = "{{ package }}-{{ version }}"
```

Comment:
will this make tags for every BSP for instance when we publish a new hal? That seems a bit excessive

Reply:
Yeah, precisely. While I agree that we're going to end up with a lot of tags in the long run, @ianrrees has convinced me that tagging every version of every crate we publish is pretty much necessary. Quoting our Matrix convo:

> yeah, I think it's the right thing to do.  Otherwise, reproducibility is about impossible for any of the crates except the HAL

Unfortunately tags aren't searchable in the github webapp (at least not that I'm a aware of), but ultimately having a lot of tags costs nothing, and since they follow a predictable naming scheme, any specific tag should be pretty easy to track down. The currently existing tags (v0.17 et al.) don't follow that convention, but it's simple to go back and retag them with the right name.

Reply:
Once thing I just noticed: while tags aren't searchable, github releases are, which release-plz creates in parallel with tags.

Reply:
Fair enough, just wanted to ensure it was properly considered. It's not a big deal.